src/__tests__/permisions.js

```diff
@@ -0,0 +1,150 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../index';
+import { unauthorizedToken, authorizedToken } from '../__mocks__/testVariables';
+
+
+chai.use(chaiHttp);
+
+const { assert } = chai;
+
+describe('Set Role Permissions', () => {
+  let unauthorised, invalidRoleId, invalidResourceId, validInput, validInput2;
+  beforeEach(() => {
+    unauthorised = {
```

Comment:
Nice Work!
I suggest you have the `unauthorised, invalidRoleId, invalidResourceId, validInput, validInput2` object in a separate file and import them here  for separation of concern

Reply:
Alright

Reply:
Done